README.md

```diff
@@ -140,6 +140,14 @@ with tools that consume JSON.
 Use `edm4hep2json --help` to see the available options for selecting which parts
 of the input file should be part of the output.
 
+## Example files
+
+Example EDM4hep files can be obtained from the Continuous Integration (CI)
+workflows. From the EDM4hep github page, go to Actions -> Key4hep build, click
+one of the runs (the latest scheduled is preferred) and they will appear at the
+bottom, under Artifacts.
```

Comment:
Are these publicly visible in the end?

Reply:
Even without github account
![screenshot_2024-08-28_1](https://github.com/user-attachments/assets/d33f87b0-720f-4c7c-a337-9f2e103b3ba7)

Reply:
nice :) Thanks for checking.

Reply:
Very nice. Thanks!